lib/svgo/js2svg.js

```diff
@@ -64,15 +66,26 @@ function JS2SVG(config) {
     this.config.indent = '    ';
   }
 
+  if (typeof this.config.eol === 'string' && this.config.eol) {
+    const configuredEOL = this.config.eol.toLowerCase();
+    if (configuredEOL === 'lf') {
```

Comment:
toLowerCase probably also not necessary. Will conflict with types anyway. Btw type and null checks will not be necessary then at all

Reply:
Addressed in 21bb65f.